src/mapclient/view/workflow/workflowwidget.py

```diff
@@ -700,6 +700,12 @@ def _create_menu_items(self):
         self._set_action_properties(self.action_New, 'action_New', self.new, 'Ctrl+N', 'Create a new Workflow')
         self.action_Open = QtGui.QAction('&Open', menu_file)
         self._set_action_properties(self.action_Open, 'action_Open', self.open, 'Ctrl+O', 'Open an existing Workflow')
+        self.action_Import_CFG = QtGui.QAction('Import Workflow Configuration', menu_workflow)
```

Comment:
Can we change these names to `Import Config.` to make them shorter and leave the tooltip to give further details.

Reply:
I've just added these changes.